### DISCUSSION TITLE
Welcome to Laragon Discussions!

### DISCUSSION THREAD

**Discussion by leokhoa** [leokhoa/laragon#344] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kasi-gajavalli-epietch** [comment]
Hi Leokhoa,

I recently migrated to laragon from WSL in windows. I am trying to install Ghostscript with imagik. As of now, my attempts to combine the two are unsuccessful. 
My unresolved errors are:

Error during generation FailedToExecuteCommand `"gs" -sstdout=%stderr -dQUIET -dSAFER -dBATCH -dNOPAUSE -dNOPROMPT -dMaxBitmap=500000000 -dAlignToPixels=0 -dGridFitTT=2 "-sDEVICE=pngalpha" -dTextAlphaBits=4 -dGraphicsAlphaBits=4 "-r300x300" -dPrinted=false -dFirstPage=1 -dLastPage=1 "-sOutputFile=C:/Users/KGAJAV~1/AppData/Local/Temp/magick-htYxuLlyVV08DrALadQzlhr0nSu_mjyT%d" "-fC:/Users/KGAJAV~1/AppData/Local/Temp/magick-0FA5MxRLQrS5aTT_cJ_Ka6zEJJjj8R71" "-fC:/Users/KGAJAV~1/AppData/Local/Temp/magick-XTisWUVi04efPAm1xqzD_wpJ420TyOQJ"' (Le fichier sp�cifi� est introuvable.
) @ error/delegate.c/ExternalDelegateCommand/516

However, gs is installed globally. Can you guide me to install it in Laragon? Thanks in advance for your input.

Sincerely
Kasi GAJAVALLI

**Reply by nessimabadi** [reply]
Hi @kasi-gajavalli-epietch , did you manage to fix this?

Thanks

**Comment by furopi** [comment]
Ty for going live on github! Keep up the good work, I just simply love Laragon!!!! ❤️

**Comment by klausveliu** [comment]
Ability to run different version of PHP to different project at the same time would be a nice feature.
Thumbs up for the time you have spend to give us these nice tool!

**Comment by vanjasavic** [comment]
Will there be a Mac version?

**Comment by trananhmanh89** [comment]
So forum will be deprecated? Hope all guides will be moved to github or doc site.
Btw, thanks for your hard work @leokhoa

**Comment by wikolopadm** [comment]
Dear Sir, Khoa Leokhoa,

I found your Laragon application extraordinary, yet I have been in technical computing since the TRS80, the Macintosh SE. I'm 40 years old
in technical computing this year and I chose you for the quality of your tool, it is exceptional.
I developed a solution in the form of scripts (.cmd .vbs .reg)
which manages the internet in the line of the iPhone, this one encrypts the internet line with a digital certificate of the APN of the operator.
At first, I installed LARAGON to test, then I saw its design quality.
You have created a great program!
I put myself at 100% to improve your tool where I could intervene.
I installed NGROK in script, it's a very good tool, as well as the coreDNS and others that I configured.
I would like to have the honor to talk to you, I let you look at my runRGEip script so that you can integrate it under laragon/bin/server-iPhone
I originally created this so that my HP I7 PC could be hosted. I host my own data through my iPhone It doesn't exist, I'm the only one.
Let's take the time to meet, I live in Geneva, Switzerland, I am a former Oracle DBA for various banks and insurance companies in Geneva.
I lost a daughter in a terrible accident, and I'm available. to resume activity.
The purpose of its modules is to pass the internet services of LARAGON in a variable which is in httpd.config (big programming with DIG)

Best regards and my respect, Mr Khoa Leokhoa,
Laurent Perret
sysadmsy7@gmail.com
0041762211389

https://github.com/FreeQI-Invest-for-smart-phone-server
https://github.com/FreeQI-Invest-for-smart-phone-server/runREGip

**Reply by wikolopadm** [reply]
Hello Leokhoa,
My name is Lorenzo.
 
if you give me your env variable for Laragon.
I can give you my .profile
 
can you do the same:
 
@reg export "HKEY_LOCAL_MACHINE\SYSTEM\ControlSet001\Control\Session Manager\Environment" "C:\environ-laragon".%date%."reg" /y
 
I still have a lot of surprises!
 
Laragon works on my laptop through my iphone in a Secure APN, it's amazing!
0.- U$
 
we need to talk, I only speak French, so here I can translate in your language and mine.

**Comment by BiggerLimits** [comment]
### THANK YOU.    THANK YOU.     THANK YOU.    THANK YOU.     THANK YOU.     THANK YOU.     THANK YOU.   

It could be said a million times more, and that would still be insufficient.

**Reply by furopi** [reply]
> ### THANK YOU. THANK YOU. THANK YOU. THANK YOU. THANK YOU. THANK YOU. THANK YOU.
> It could be said a million times more, and that would still be insufficient.

Just for what exactly I wonder... For creating Laragon or for putting the stuff on github? Both are awesome milestones imho... <3 (Assuming both)

**Comment by metacraft-hq** [comment]
Wow! I am super impressed - will promote [laragon.org](https://laragon.org/) across my platforms

**Comment by sedoundele** [comment]
hello dear all i need help please i am using laragon version 5.0 and i want to know; How to install PHP’s Oracle OCI extension on laragon

**Comment by ulul363** [comment]
I regret to inform you that due to financial difficulties, I cannot keep the self-hosted forum and moving our discussions to GitHub.

https://github.com/leokhoa/laragon/discussions

I understand that this may cause inconvenience and I apologize for any disruption this may cause.

I hope to continue our discussions and exchange of ideas on GitHub and I look forward to your participation there.

If you wonder about the future of Laragon, I have made a story about "[How I Created Laragon and the Current Status of Laragon’s Development](https://medium.com/@leovn/how-i-created-laragon-and-the-current-status-of-laragons-development-bc927c5ce1d6)"

I appreciate your understanding and support.

Sincerely,

Le Ngoc Khoa.

how to this solution?

**Comment by jaisimul** [comment]
Can anyone please help to do room visualzation using AIML python or opencv, We needs px to px sgementation and can replace that masked report to new texture like on floor

**Comment by nzenongbraxton** [comment]
hey

**Comment by zoulkoro** [comment]
hi zoulkorneni, please i just want to know how change my php. i actually have 8.1 version and i need the 8.2.17 version
